solr/core/src/test/org/apache/solr/update/processor/ParsingFieldUpdateProcessorsTest.java

```diff
@@ -819,7 +819,7 @@ public void testFailedParseMixedBoolean() throws Exception {
       if (longVal == o) {
         foundLong = true;
       } else {
-        assertThat(o, IS_STRING);
+        assertTrue(o instanceof String);
```

Comment:
I suspect you are trying to minimize any refactoring...  However, could you eliminate IS_STRING?   A quick glance makes me think it is only used in asserts...

Reply:
Yea there were a bunch more usages of `IS_...` I can clean them up too.